### DISCUSSION TITLE
autodiscoverFilter doesn't pickup the .github repo

### DISCUSSION THREAD

**Discussion by lmilbaum** [renovatebot/renovate#25511] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, select which platform you are using.

github.com

### Was this something which used to work for you, and then stopped?

I am trying to get this working for the first time

### Describe the problem

The following filter doesn't pick up the `.github` repo:
```
  autodiscoverFilter: ["sagano-org/*"]
```

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

### Have you created a minimal reproduction repository?

Placeholder value, please select the correct response from the dropdown

**Comment by rarkins** [comment, marked as answer]
Fix in #25516
